### ISSUE TITLE
Cannot install in pip3.11

### ISSUE THREAD

**Issue by andrea-cassioli-maersk** [google/yggdrasil-decision-forests#44]
Hi I am trying to install `tensorflow_decision_forests` on my Mac for python3.11 and got the following error

```
➜  ML-COLGEN-Hackathon-June2023 git:(main) ✗ pip3.11 install tensorflow_decision_forests --upgrade
ERROR: Could not find a version that satisfies the requirement tensorflow_decision_forests (from versions: none)
ERROR: No matching distribution found for tensorflow_decision_forests

```

I use

```
➜  yggdrasil-decision-forests git:(main) ✗ pip3.11 --version
pip 23.1.2 from /opt/homebrew/lib/python3.11/site-packages/pip (python 3.11)

```

any clue? It works nicely in pip3.10

**Comment by andrea-cassioli-maersk** [comment]
btw in https://pypi.org/project/tensorflow-decision-forests/ the instruction on the top says

```
pip install tensorflow-decision-forests
```

but your docs says 

```
pip install tensorflow_decision_forests
```

**Comment by rstz** [comment]
Hi Andrea,
we simply haven't released TF-DF for Python 3.11 because the release script wasn't configured for it :)
Python 3.11 will be available with the next TF-DF version. 

I'll try to compile a TF-DF package for 3.11 now and see if it works quickly - if everything just works, I'll upload it to PyPi today, but no promises :)

Thank you for reporting the inconsistency in the installation instructions - it looks like Pypi resolves both to TF-DF so both are fine.

**Comment by andrea-cassioli-maersk** [comment]
thanks @rstz for the quick response!

**Comment by rstz** [comment]
Update: TF-DF is now [available for Python 3.11 on PyPi](https://pypi.org/project/tensorflow-decision-forests/#files)
